src/main/java/com/github/tomakehurst/wiremock/standalone/CommandLineOptions.java

```diff
@@ -396,6 +400,9 @@ public CommandLineOptions(String... args) {
       stores.getSettingsStore().set(newSettings);
     }
 
+    TemplateEngine templateEngine =
```

Comment:
Thinking we'd be better off initialising this inside `FilenameMaker` and passing the template as a string constructor parameter.